README.md

```diff
@@ -14,9 +14,17 @@
 
 - [Installation](#installation)
 - [Usage](#usage)
-  - [Signed URLs](#signed-urls)
-  - [Srcset Generation](#srcset-generation)
+- [Signed URLs](#signed-urls)
+- [Srcset Generation](#srcset-generation)
+  - [Fixed Width Images](#fixed-width-images)
+    - [Variable Quality](#variable-quality)
+  - [Variable Size Images](#variable-size-images)
+    - [Custom Widths](#custom-widths)
+    - [Minimum and Maximum Width Ranges](#minimum-and-maximum-width-ranges)
+    - [Width Tolerance](#width-tolerance)
+    - [Explore Target Widths](#explore-target-widths)
```

Comment:
I'm wondering if this should be its own item under `Srcset Generation`? It is related to fluid-width srcsets but it isn't a subset of that functionality. Thoughts?

Reply:
Yeah, I think it probably should have been. I'll add some `#` to the title there.